content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/resupply_warehouses.rst

```diff
@@ -2,69 +2,166 @@
+an inter-warehouse transfer using replenishment.
+
+There are two configuration methods for inter-warehouse replenishment:
+
+#. Make to order (MTO)
+#. Reordering rule
+
+.. seealso::
+   :doc:`Difference between MTO and reordering rules <../replenishment>`
 
 Configuration
 =============
 
-To resupply from another warehouse, first go to :menuselection:`Inventory --> Configuration -->
-Settings --> Warehouse` and activate :guilabel:`Multi-Step Routes`. Then, click :guilabel:`Save` to
-apply the setting.
+To resupply from another warehouse, first go to :menuselection:`Inventory app --> Configuration -->
+Settings`. In the :guilabel:`Warehouse` section, activate :guilabel:`Storage Locations`. Then, click
+:guilabel:`Save` to apply the setting.
 
-.. image:: resupply_warehouses/virtual-warehouses-settings.png
+.. image:: resupply_warehouses/storage-locations.png
    :align: center
-   :alt: Enable Multi-Step Routes in Inventory settings.
+   :alt: Enable Storage Locations in Inventory settings.
 
-View all the configured warehouses by going to :menuselection:`Inventory --> Configuration -->
-Warehouses`.
+Warehouses
+----------
 
-Create a new warehouse by clicking :guilabel:`Create`. Then, give the warehouse a name and a
+Configure the settings for the central warehouse and retail shops by going to
+:menuselection:`Inventory app --> Configuration --> Warehouses`.
+
+Each central warehouse and retail shop must have its own warehouse.
+
+Create a new warehouse by clicking :guilabel:`New`. Then, give the warehouse a name and a
 :guilabel:`Short Name`. Finally, click :guilabel:`Save` to finish creating the warehouse.
 
 After that, go back to the :guilabel:`Warehouses` page and open the warehouse that will be
-resupplied by the second warehouse. Then, click :guilabel:`Edit`. In the :guilabel:`Warehouse
-Configuration` tab, locate the :guilabel:`Resupply From` field, and check the box next to the
-second warehouse's name. If the warehouse can be resupplied by more than one warehouse, make sure
-to check those warehouses' boxes too. Finally, click :guilabel:`Save` to apply the setting. Now,
-Odoo knows which warehouses can resupply this warehouse.
-
-.. image:: resupply_warehouses/resupply-from-second-warehouse.png
+resupplied by the central warehouse. In the :guilabel:`Warehouse Configuration` tab, locate the
+:guilabel:`Resupply From` field, and check the box next to the central warehouse's name. If the
+warehouse can be resupplied by more than one warehouse, make sure to check those warehouses' boxes
+too. Finally, click :guilabel:`Save` to apply the setting. Now, Odoo knows which warehouses can
+resupply this warehouse.
+
+.. example::
+   The central warehouse that will supply the shops is called `Central warehouse`. The
+   :guilabel:`Resupply From` field is set to this warehouse on the shop's warehouse configuration
+   page.
+
+.. image:: resupply_warehouses/warehouse.png
```

Comment:
I think this image should go inside the example (since the text in the example references the image)-- it should be indented further to be nested inside example!

Reply:
Nice catch!!!